test/conformance/tests/test_sriov_operator.go

```diff
@@ -90,10 +91,10 @@ var _ = Describe("[sriov] operator", Ordered, func() {
 				Expect(err).ToNot(HaveOccurred())
 
 				Expect(len(selectedNodes)).To(BeNumerically(">", 0), "There must be at least one worker")
-				candidate := selectedNodes[0]
-				candidate.Labels["sriovenabled"] = "true"
-				_, err = clients.CoreV1Interface.Nodes().Update(context.Background(), &candidate, metav1.UpdateOptions{})
+				patch := []byte(`{"metadata":{"labels":{"sriovenabled":"true"}}}`)
+				candidate, err := clients.CoreV1Interface.Nodes().Patch(context.Background(), selectedNodes[0].Name, types.StrategicMergePatchType, patch, metav1.PatchOptions{})
```

Comment:
nit: technically we dont need strategic merge and the regular mergePatchType is enough IMO

Reply:
i dont have any objections to keep this so lgtm